homeassistant/components/homewizard/coordinator.py

```diff
@@ -45,6 +44,19 @@ async def _async_update_data(self) -> DeviceResponseEntry:
             raise UpdateFailed("Device did not respond as expected") from ex
 
         except DisabledError as ex:
+            async_create_issue(
+                self.hass,
+                DOMAIN,
+                f"api_disabled_{self.config_entry.entry_id if self.config_entry is not None else ''}",
+                data={CONF_IP_ADDRESS: self.api.host, CONF_DEVICE_NAME: self.name},
+                is_fixable=True,
+                learn_more_url="https://www.home-assistant.io/integrations/homewizard/#enable-the-api",
```

Comment:
The 'learn_more' does not show up in a repair flow. What is the intended behaviour?